helm/chaos-mesh/templates/_helpers.tpl

```diff
@@ -113,5 +113,30 @@ Define the webhook's name
 Define the prefix of
 */}}
 {{- define "registry-prefix" -}}
-{{if .Values.registry}}{{.Values.registry}}/{{end}}
+{{if .Values.images.registry}}{{.Values.images.registry}}/{{end}}
```

Comment:
As [this best practice](https://helm.sh/docs/chart_best_practices/templates/#formatting-templates) suggested, we'd better to format these templates with some whitespaces :)

Reply:
updated in 6af6e041, PTAL! :)